gravityflow/gravityflowformconnector
# Add Gravity_Flow_Merge_Tag_Form_Submission_Url

This PR adds the Gravity_Flow_Merge_Tag_Form_Submission_Url class so the merge tag will be processed in the Gravity Forms confirmations.

Testing Instructions
- This requires Gravity Forms 2.3.3.10 which contains the gform_pre_handle_confirmation action.
- It also requires [this PR](https://github.com/gravityflow/gravityflow/pull/164) for Gravity Flow.
- Add a form submission step as the first step in the workflow and assign it to an email field.
- Add the assignee attribute to the merge tag {workflow_form_submission_link: assignee=email_field|3}
- Ensure that the link works as expected.
- Remove the assignee attribute and submit while logged in as the assignee of the first step. The token="true" attribute must be set for non-email assignees for the token to be added. Ensure that the URL works as expected.
- Check for regression issues.
